mlflow/server/js/src/components/RunView.js

```diff
@@ -156,6 +193,10 @@ class RunView extends Component {
             </div>
             : null
           }
+          <div className="run-info">
+            <span className="metadata-header">Run ID: </span>
+            <span className="metadata-info">{run.getRunUuid()}</span>
+          </div>
```

Comment:
Put this farther up in the list, after Date. It should be in a consistent place for all runs since people will often copy-paste it.